### PR TITLE
Added error handling to axios calls

### PR DESCRIPTION
**Pull request by mKleinCreative**

### CODE REVIEW

File: source/server/controllers/v1/projectController.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ const handleGetProjectsByUserId = ( request, response ) => {
     .catch( error => {
       console.log( chalk.red('There was an error in handleGetProjectsByUserId: ')) // eslint-disable-line no-console
       console.trace(error) // eslint-disable-line no-console
-      response.status( 400 ).send( { Error: error } )
+      response.status( 400 ).json( { Error: error } )
```
Comment (nodatall) [inline source/server/controllers/v1/projectController.js]: Could you change this is all the rest of the controllers?

File: source/components/Projects/ProjectListContainer.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import React, { Component } from 'react'
 import axios from 'axios'
+import handyPerson from '../handyPerson'
```
Comment (nodatall) [inline source/components/Projects/ProjectListContainer.js]: Lets call it `componentErrorHandler`

File: source/components/Projects/ProjectListContainer.js [hunk 3]
```diff
@@ -10,14 +11,17 @@ export default class ProjectListContainer extends Component {
...
 
-  componentWillMount = () => {
+  componentDidMount = () => {
     axios.get(`http://localhost:1337/user/${this.state.userId}/projects`)
       .then( body => {
         globalState.set({ projects: body.data })
       })
+      .catch( error => {
+        handyPerson( 'ProjectListContainer', error)
+      })
```
Comment (nodatall) [inline source/components/Projects/ProjectListContainer.js]: One line this and take out curlies:
```js
.catch( error => componentErrorHandler( 'ProjectListContainer', error  )
```

File: source/components/TextField/TextFieldContainer.js [hunk 4]
```diff
@@ -40,6 +41,9 @@ export default class TextFieldContainer extends Component {
         globalState.set({ [type]: updatedState })
         this.setState({ editing: false })
       })
+      .catch( error => {
```
Comment (nodatall) [inline source/components/TextField/TextFieldContainer.js]: One line this as well.

**Comment by mKleinCreative** [comment]
Updated @nodatall

**Pull request merged**